src/Microsoft.VisualStudio.ProjectSystem.Managed/Threading/Tasks/SequentialTaskExecutor.cs

```diff
@@ -17,9 +17,18 @@ internal sealed class SequentialTaskExecutor : IDisposable
         private Task _taskAdded = Task.CompletedTask;
         private readonly object _syncObject = new();
         private readonly CancellationTokenSource _disposedCancelTokenSource = new();
+        private readonly JoinableTaskCollection _joinableCollection;
+        private readonly JoinableTaskFactory _joinableFactory;
+
+        public SequentialTaskExecutor(JoinableTaskContextNode joinableTaskContext)
+        {
+            _joinableCollection = joinableTaskContext.CreateCollection();
+            _joinableCollection.DisplayName = nameof(SequentialTaskExecutor);
```

Comment:
What about including the display name as a parameter to the constructor? If we're going to use `SequentialTaskExecutor` in multiple scenarios it may be helpful to distinguish one usage from another.

Reply:
Good idea. Done.